testdata-gen/src/main.rs

```diff
@@ -134,13 +134,13 @@ enum SubCommand {
     SetRedeemPeriod(SetRedeemPeriodInfo),
     /// Set replace period.
     SetReplacePeriod(SetReplacePeriodInfo),
-    /// Transfer DOT collateral
+    /// Transfer collateral.
     FundAccounts(FundAccountsInfo),
 }
 
 #[derive(Clap)]
 struct SetExchangeRateInfo {
-    /// Exchange rate from BTC to DOT.
+    /// Exchange rate (i.e. Planck per Satoshi).
```

Comment:
We should specify the unit. The actual function call is in dot per btc, not planck per satoshi. But we divide this integer by 100_000 to get a ratio, so technically both are wrong.